ietf/ietfauth/validators.py

```diff
@@ -0,0 +1,34 @@
+# Copyright The IETF Trust 2024, All Rights Reserved
+import re
+
+from django import forms
+from django.conf import settings
+from django.core.exceptions import ValidationError
+
+
+def prevent_at_symbol(name):
+    if "@" in name:
+        raise forms.ValidationError(
+            "Please fill in name - this looks like an email address (@ is not allowed in names)."
+        )
+
+
+def prevent_system_name(name):
+    name_without_spaces = name.replace(" ", "").replace("\t", "")
+    if "(system)" in name_without_spaces.lower():
+        raise forms.ValidationError("Please pick another name - this name is reserved.")
+
+
+def prevent_anonymous_name(name):
+    name_without_spaces = name.replace(" ", "").replace("\t", "")
+    if "anonymous" in name_without_spaces.lower():
+        raise forms.ValidationError("Please pick another name - this name is reserved.")
+
+
+def is_allowed_address(value):
+    """Validate that an address complies with datatracker requirements"""
+    for pat in settings.EXCLUDED_PERSONAL_EMAIL_REGEX_PATTERNS:
+        if re.search(pat, value):
+            raise ValidationError(
+                "This email address is not valid in a datatracker account"
```

Comment:
I realize this is old, but is it conflating User.username concerns with Email.address concerns? If so, we should clarify.

Reply:
I think this is doing things like preventing `some-wg-list@ietf.org` from being signed up as an email. I don't think it's for the username concerns.